api/rest/.htaccess

```diff
@@ -0,0 +1,4 @@
+RewriteEngine On
+RewriteCond %{REQUEST_FILENAME} !-f
+RewriteRule ^ index.php [QSA,L]
```

Comment:
Done.  Added a comment that references http://docs.slimframework.com/routing/rewrite/